src/main/webapp/app/exercises/programming/manage/services/programming-exercise-participation.service.ts

```diff
@@ -185,4 +185,10 @@ export class ProgrammingExerciseParticipationService implements IProgrammingExer
     retrieveCommitHistoryForTemplateSolutionOrTests(exerciseId: number, repositoryType: string): Observable<CommitInfo[]> {
         return this.http.get<CommitInfo[]>(`${this.resourceUrl}${exerciseId}/commit-history/${repositoryType}`);
     }
+
+    retrieveCommitHistoryForAuxiliaryRepository(exerciseId: number, repositoryType: string, auxiliaryRepositoryId: number): Observable<CommitInfo[]> {
```

Comment:
Do we need the `repositoryType` here? It looks unused

Reply:
Am I missing something? I think it gets used, in line 192, at the very end before the params
```
        return this.http.get<CommitInfo[]>(`${this.resourceUrl}${exerciseId}/commit-history/${repositoryType}`, { params: params });
```

Reply:
Sorry, I was looking at the wrong method. Would you say that `repositoryType` is required as a parameter here given the method name `retrieveCommitHistoryForAuxiliaryRepository`? I feel like it would only ever be `AUXILIARY` here

Reply:
aah I see yea, it's indeed not needed here :+1: